azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -21,16 +24,118 @@
         DataflowProfileOperations,
     )
 
+console = Console()
 
 class DataFlowProfiles(Queryable):
     def __init__(self, cmd):
         super().__init__(cmd=cmd)
+        self.instances = Instances(cmd=cmd)
         self.iotops_mgmt_client = get_iotops_mgmt_client(
             subscription_id=self.default_subscription_id,
         )
         self.ops: "DataflowProfileOperations" = self.iotops_mgmt_client.dataflow_profile
         self.dataflows = DataFlows(self.iotops_mgmt_client.dataflow)
 
+    def create(
+        self,
+        name: str, 
+        instance_name: str, 
+        resource_group_name: str, 
+        profile_instances: Optional[int] = 1, 
+        log_level: Optional[str] = 'info', 
```

Comment:
nit, styling wise use double quotes around info here. You can install the `black` python formatter to streamline this with autoformatting. See dev_requirements.

Same deal for the value in commands_dataflow.

Reply:
do you update any of the format settings? this resulted in a lot of changes for some of the files

Reply:
Yes use line length 119

Reply:
```    
"black-formatter.args": [
        "--line-length=119"
    ]
```